src/Master.cxx

```diff
@@ -2,11 +2,13 @@
 #include "CommandHandler.h"
 #include "FileWriterTask.h"
 #include "Source.h"
+#include "commandproducer.h"
 #include "logger.h"
 #include <chrono>
-// #include "helper.h"
-// #include "utils.h"
-#include "commandproducer.h"
+#include <rapidjson/document.h>
```

Comment:
Interesting, I haven't come across such a policy. It is difficult to distinguish between headers which are unused because they are included elsewhere or unused because they are completely unneeded, therefore I'd rather remove them. Do you prefer to keep them?